subprojects/logging/src/main/java/org/gradle/internal/featurelifecycle/DeprecatedUsageProgressDetails.java

```diff
@@ -52,6 +52,9 @@
     @Nullable
     String getContextualAdvice();
 
+    @Nullable
```

Comment:
can you add some javadoc to this (like the other methods) and a `@since 6.2` so it's clear this part came later?

Reply:
Will do!